sdk/src/main/scala/com/horizen/forge/AbstractForgeMessageBuilder.scala

```diff
@@ -97,7 +97,7 @@ abstract class AbstractForgeMessageBuilder[
 
       val forgingResult = eligibleForgerOpt
         .map { case (forgingStakeMerklePathInfo, privateKey25519, vrfProof, _) =>
-          forgeBlock(nodeView, nextBlockTimestamp, branchPointInfo, forgingStakeMerklePathInfo, privateKey25519, vrfProof, timeout)
+          forgeBlock(nodeView, nextConsensusEpochNumber, nextBlockTimestamp, branchPointInfo, forgingStakeMerklePathInfo, privateKey25519, vrfProof, timeout)
```

Comment:
let's keep it as it was before.
we can reconstruct `nextConsensusEpochNumber` from the `nextBlockTimestamp`

Reply:
done